app/src/main/java/com/example/mytask/presentation/UserAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.mytask.presentation
+
+import android.annotation.SuppressLint
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.mytask.data.entities.Teacher
+import com.example.mytask.databinding.FragmentViewInfoTeacherBinding
+
+class UsersAdapter : RecyclerView.Adapter<UsersAdapter.UsersViewHolder>() {
```

Comment:
Here should be RecyclerView.ViewHolder as a parameter of generic.

Reply:
but it implement for Adapter, why should be ViewHolder